src/app/redux/GlobalReducer.js

```diff
@@ -120,7 +120,7 @@ export default function reducer(state = defaultState, action = {}) {
             return payload.pinnedPosts.reduce((acc, pinnedPost) => {
                 const author = pinnedPost.get('author');
                 const permlink = pinnedPost.get('permlink');
-                return state.updateIn(
+                return acc.updateIn(
```

Comment:
This fixes a bug with marking pinned posts as seen. Good catch!

Reply:
Thanks for the help!